Dockerfile

```diff
@@ -26,6 +26,11 @@ FROM python:2.7
 #
 # set up environment
 #
+ENV TINI_VERSION v0.10.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
```

Comment:
Is there a good executive summary somewhere of the benefits of using `tini` as the container pid 1?  I'm aware vaguely that it solves issues with zombie processes not being correctly reaped, but I don't have a clear sense of concrete cases where it fixes issues.